code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -948,3 +948,30 @@
 				to_chat(M, "<span class='warning'>Your missing arm aches from wherever you left it.</span>")
 				M.emote("sigh")
 	return ..()
+
+	/datum/reagent/iodomethane
+		name = "Methyl Iodide"  //rocket fuel precursor time
+		id = "iodomethane"
+		description = "A potent toxin created by adding iodine to methanol and phosphorus."
+		color = "#00000000" // rgb: 0, 0, 0, 0
+		taste_description = "water"
+		taste_mult = 0
+		metabolization_rate = 0.6 * REAGENTS_METABOLISM
+		var/toxpwr = 4.5
+
+	/datum/reagent/methanol
+		name = "Methanol"
+		id = "methanol"
+		description = "Quite flammable and mildly toxic."
+		color = "#00000000"
+		taste_description = "chilli peppers and misery"
+		var/toxpwr = 0.25
+
+	/datum/reagent/dimethylmercury
+		name = "Dimethylmercury"  //Rocket fuel time
+		id = "dimethylmercury"
+		description = "An extremely potent toxin and explosive declared too dangerous for use even as a rocket fuel."
+		color = "#00000000" // rgb: 0, 0, 0, 0
+		taste_description = "sweet, sweet death"
+		metabolization_rate = 0.7 * REAGENTS_METABOLISM
+		var/toxpwr = 9
```

Comment:
Note that since this reagent basically explodes at even 5 degrees above room temperature, its high toxpwr won't matter unless either the explosion reaction is tweaked or the mob is suffering from hypothermia.